Source/Scripts/SyntaxAstVisitor.cs

```diff
@@ -172,12 +158,12 @@ protected static void VisitType(Type type, Stack<Type> toVisit) {
   }
 
   /// <summary>
-  /// Return all field/property names appearing in <see cref="RedundantField"/>
+  /// Return all field/property names appearing in <see cref="SyntaxBaseType"/>
   /// attributes of the specified type (or its base types).
```

Comment:
```suggestion
  /// Return the argument of the <see cref="SyntaxBaseType"/> attribute on the specified type if present,
  /// or its normal base type otherwise.
```